mibs/sensorgateway/sensorgateway.mib

```diff
@@ -0,0 +1,1549 @@
+--SensorGateway control MIB.
+--
+--Author                   Date        Comment
+--==================================================================
+--SiebenSense		    Mar 29, 2012  Rev. D

```

Comment:
What is the license on this file?

Reply:
There is none, so public domain.

Reply:
That's not the way it works, it's copyright to someone by default and has been for over a century.

Reply:
Depending on jurisdiction, a factual collection of data can not have a copyright.

What's the proposed way forward, then?

Reply:
Either we determine that this is legal, so it can go in the main SNMP repo or we determine that it isn't and abandon it.